### ISSUE TITLE
validation accuracy

### ISSUE THREAD

**Issue by sean-zhuh** [djiajunustc/TransVG#19]
Hi, I've trained TransVG for 30 epochs, where at the first 20 epochs, it only reaches 67 acc@0.5, is it normal? could you please provide the validation accuracy log file to me, i.e., the accuracy on val set of RefCOCO at each epoch. My e-mail is sean.zhuh@gmail.com.

Thanks a lot.

**Comment by djiajunustc** [comment]
In general, it takes more training epochs for the training of transformers to converge. I've trained for 90 epochs on RefCOCO. Don't worry about it.
